src/component/playground/Playground/PlaygroundGuidance/PlaygroundGuidance.tsx

```diff
@@ -0,0 +1,44 @@
+import { Typography, Box, Divider } from '@mui/material';
+import { PlaygroundGuidanceSection } from './PlaygroundGuidanceSection/PlaygroundGuidanceSection';
+
+export const PlaygroundGuidance = () => {
+    return (
+        <Box sx={{ ml: 4 }}>
+            <Typography variant="body1">
+                Unleash playground is for helping you to undestand how unleash
+                works, how feature toggles are evaluated and for you to easily
+                debug your feature toggles.
+            </Typography>
+
+            <Divider sx={{ mt: 2, mb: 2 }} />
+
+            <Typography variant="body1" sx={{ mb: 1 }}>
+                What you need to do is:
+            </Typography>
+
+            <PlaygroundGuidanceSection
+                headerText="  Select in which environment you want to test your
+                            feature toggle configuration"
+                bodyText="  You can also specify specific projects, or check
+                            toggles in all projects."
+                sectionNumber="1"
+            />
+
+            <PlaygroundGuidanceSection
+                headerText="Select a context field that you'd like to check"
+                bodyText="Configure multiple context fields or leave empty to test against an empty context"
```

Comment:
When you test against an empty context, we automatically apply `appName`, though. Should that be mentioned?

```suggestion
                bodyText="You can configure as many context fields context fields as you want. You can also leave the context empty to test against an empty context."
```